client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -365,38 +408,43 @@ class TransferDomainPrecheck extends React.PureComponent {
 
 		return (
 			<div className="transfer-domain-step__precheck">
-				{ this.getHeader() }
-				{ this.getStatusMessage() }
-				{ this.getPrivacyMessage() }
-				{ this.getEppMessage() }
-				<Card className="transfer-domain-step__continue">
-					<div className="transfer-domain-step__continue-text">
-						<p>
-							{ translate(
-								'Note: These changes can take some time to take effect. ' +
-									'Need help? {{a}}Get in touch with one of our Happiness Engineers{{/a}}.',
-								{
-									components: {
-										a: (
-											<a
-												href={ support.CALYPSO_CONTACT }
-												rel="noopener noreferrer"
-												target="_blank"
-											/>
-										),
-									},
-								}
-							) }
-						</p>
+				{ this.transferIsRestricted() && this.getTransferRestrictionMessage() }
```

Comment:
Feels weird.

Let's do an `if` above and `return` if restricted, otherwise the old thing.

Reply:
Sounds good.